android/sdk/src/main/java/org/jitsi/meet/sdk/AppInfoModule.java

```diff
@@ -64,6 +64,9 @@ public AppInfoModule(ReactApplicationContext reactContext) {
 
         Map<String, Object> constants = new HashMap<>();
 
+        constants.put(
+            "buildNumber",
+            packageInfo == null ? "" : ""+packageInfo.versionCode);
```

Comment:
Why ```""+packageInfo.versionCode```? If it's supposed to be a String conversion, can we use some better approach? And I think there must be a space around the ```+```

Reply:
This is the most concise way I know, I picked it up from some other project, not sure. I'll fixup the spaces.